fakestorage/object.go

```diff
@@ -19,14 +19,15 @@ import (
 
 // Object represents the object that is stored within the fake server.
 type Object struct {
-	BucketName  string `json:"-"`
-	Name        string `json:"name"`
-	ContentType string `json:"contentType"`
-	Content     []byte `json:"-"`
+	BucketName      string `json:"-"`
+	Name            string `json:"name"`
+	ContentType     string `json:"contentType"`
+	ContentEncoding string `json:"contentEncoding"`
+	Content         []byte `json:"-"`
 	// Crc32c checksum of Content. calculated by server when it's upload methods are used.
 	Crc32c  string            `json:"crc32c,omitempty"`
 	Md5Hash string            `json:"md5hash,omitempty"`
-	ACL     storage.ACLEntity `json:"acl,omitempty"`
+	ACL     []storage.ACLRule `json:"acl,omitempty"`
```

Comment:
Since fake-gcs-server is also used as a library, could we avoid a breaking change here? Or would the previous ``ACL`` attribute not make any sense? 🤔

Reply:
the previous ACL doesn't make sense anymore, that's why I replaced it.

Reply:
it's more compatible with GCP protocol

Reply:
Yeah I was worrying about the fact that people's code will fail to compile after this change, so I wonder if we should introduce q new attribute for that.

Let me take another look at the code in a bit and check what it would take.